modules/Members/language/en_UK.json

```diff
@@ -0,0 +1,11 @@
+{
+  "members/members": "Members",
+  "members/new_members": "New members",
+  "members/no_members": "No members found.",
+  "members/member_lists": "Member Lists",
+  "members/member_list_hide_banned_users": "Hide banned users?",
+  "members/member_list_search_min_3_chars": "Please ensure your search query is at least 3 characters long.",
+  "members/member_list_search_no_results": "No user found for '{{search}}'.",
+  "members/member_list_toggled": "Successfully {{value}} {{list}} list.",
```

Comment:
Might this make sense to split into two terms (i.e. one for the positive case and one for the negative case), in case two different languages have different sentence structures in each case?

Reply:
Great point